clang/lib/Driver/Driver.cpp

```diff
@@ -1824,7 +1826,10 @@ void Driver::PrintSYCLToolHelp(const Compilation &C) const {
     }
     // do not run the tools with -###.
     if (C.getArgs().hasArg(options::OPT__HASH_HASH_HASH)) {
-      llvm::errs() << "\"" << ExecPath << "\" \"" << ToolArgs[1] << "\"\n";
+      llvm::errs() << "\"" << ExecPath << "\" \"" << ToolArgs[1] << "\"";
+      if (!ToolArgs[2].empty())
+        llvm::errs() << " \"" << ToolArgs[2] << "\"";
```

Comment:
Technically, a `while` loop would've looked cleaner than a hardcoded `[2]`. But I see why one can prefer the code this way (to avoid extra complexity in an unlikely-to-change helper function).

Reply:
I could go either way.  It was just a single argument used in a very specific output function with know limited array args.